srcpkgs/qflipper/template

```diff
@@ -1,10 +1,12 @@
 # Template file for 'qflipper'
 pkgname=qflipper
-version=1.1.0
+version=1.1.3
 revision=1
 _nanopb_version=0.4.5
+archs="~i686" #https://github.com/flipperdevices/qFlipper/issues/120
```

Comment:
How come it's buildable for arm 32bit?

Reply:
Well, because you're doing it wrong. Those addition `*-devel` packages in `hostmakedepends` is wrong!!

Reply:
Sorry, I added the `*-devel` packages to `hostmakedepends` because the cross builds on my machine were complaining about being unable to find the QT modules for those packages and that appeared to resolve it.

Reply:
Don't add `-spec` into configure args, it will be set by `build-style` and some magic needs to be done for cross compilation.
In the end of the day, it will use same spec with linux-g++